app/assets/javascripts/donations.js

```diff
@@ -0,0 +1,30 @@
+var main_height = $("main").height();
```

Comment:
Identifier 'main_height' is not in camel case.<br>'main_height' is defined but never used.